master/contrib/docker/pythonnode_worker/Dockerfile

```diff
@@ -12,7 +12,7 @@ ARG         DEBIAN_FRONTEND=noninteractive
 user root
 # Install required npm packages
 RUN  curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
-        apt-get install -y -o APT::Install-Recommends=false -o APT::Install-Suggests=false \
```

Comment:
sorry, I removed that line and did not bother to check the script.
anyway, as this should be an example for general purpose workers, it seems not so clear (at least to me). maybe mention it in the comment above?